src/hexo/source/about/FAQ.md

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
This file should not be added and is a remainder from the previous project structure.

The same goes for all the other under `src/hexo`.

Reply:
@alrra should I just delete everything under srx/hexo. This is what was causing conflicts for me on Friday that Anton had to help me with.